### PR TITLE
added positionfilter to index

### LINKED ISSUE #9: Missing Documentation for filters 51-58
The documentation for filter types 51 to 58 are still missing.

### LINKED ISSUE #13: Write index to filter documentation
We need a `Readme.md` file within the filer documentation that provides an overview as well as an index to the filter documentation. We might need something similar for the emerging virtual filters, albeit this might fit better within the general manual for now.

### PR DESCRIPTION
**Pull request by LRahlff**
filter 51- 58 are in the adapters
#9 and #13 already fixed?

### CODE REVIEW

File: Fish_Doku/Filter_Types.md
Comment (LRahlff) [inline Fish_Doku/Filter_Types.md:1]: is this file enough as index, or does it has to be an README.md?

Comment (Doralitze) [reply Fish_Doku/Filter_Types.md:1]: Maybe you should leaf it as an index and hint the existence of this file within the directories README.md.

**Review by Doralitze — CHANGES_REQUESTED** [review]
I think we should make a clear distinction between virtual filters (only handled by the GUI) and non-virtual ones (that therefore define an interface with fish). Maybe use a different directory. I  also think that these filters deserve an entry within the manual as they are quite complex to handle (albeit this does not need to be done now).

File: Fish_Doku/README.md
Comment (LRahlff) [inline Fish_Doku/README.md:1]: did you mean like this?

**Comment by LRahlff** [comment]
> I think we should make a clear distinction between virtual filters (only handled by the GUI) and non-virtual ones (that therefore define an interface with fish). Maybe use a different directory. I also think that these filters deserve an entry within the manual as they are quite complex to handle (albeit this does not need to be done now).

oh, yes i didnt know that in the editor filter were prepared too. what do you think of this structure? distinction between fish and gui and also between usage and technical information.

File: Fish_Doku/README.md
Comment (Doralitze) [reply Fish_Doku/README.md:1]: Yes.

**Review by Doralitze — APPROVED** [review]
LGTM

**Pull request merged**